app/views/foods/index.html.erb

```diff
@@ -0,0 +1,37 @@
+<p style="color: green"><%= notice %></p>
+<section class="recipes-section">
+<div class="new-recipe-link">
+  <%= link_to "New food", new_recipe_path, class: 'btn btn-primary' %>
+</div>
```

Comment:
- [ ] This button to add new food is leading to adding a new recipe. Please make sure to fix this issue. Clicking on this button should redirect users to add a new food page.